python_ta/__init__.py

```diff
@@ -188,11 +188,11 @@ def reset_linter(config=None, file_linted=None):
              'type': 'string',
              'metavar': '<pyta_reporter>',
              'help': 'Template file for html format of htmlreporter output.'}),
-        ('pyta-output-file',
-            {'default': 'output.html',
-             'type': 'string',
-             'metavar': '<pyta_reporter>',
-             'help': 'Output file for htmlreporter.'}),
+        ('pyta-save-html-report',
```

Comment:
Okay, so I'm actually thinking that we should just delete this option entirely. Whether or not to save to an output file will be based on the argument to `check_all` only, not in the `.pylintrc` file.

Also review the default `.pylintrc` file and remove the old `pyta-output-file` option, if we included it.

Reply:
that's strange, I thought I removed the `pyta-save-html-report` on the remote commit, that's my bad. Also, do you want me to remove the `pyta-output-file` option from `__init__.py` too?

Reply:
Yes, please remove the `pyta-output-file` option from this file.

Reply:
@simonchenwastaken Yes, remove the option.